android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -72,6 +72,23 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
     refreshEditableFields();
   }
 
+  @Override
+  public void onSaveInstanceState(Bundle outState)
+  {
+    super.onSaveInstanceState(outState);
+    saveEdits();
```

Comment:
Сохранение временное, поменяю название методов.


> On Mar 24, 2016, at 7:38 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/EditorFragment.java <https://github.com/mapsme/omim/pull/2506#discussion_r57346367>:
> 
> > @@ -72,6 +72,23 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
> >      refreshEditableFields();
> >    }
> >  
> > +  @Override
> > +  public void onSaveInstanceState(Bundle outState)
> > +  {
> > +    super.onSaveInstanceState(outState);
> > +    saveEdits();
> А почему так? Метод может быть вызван, если, например, юзеру позвонили и фрагмент убился. Или это только временное сохранение?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2506/files/fdf2eabe22ea83dc51ce2bb2c714243bda45233d#r57346367>